app/App.js

```diff
@@ -4,15 +4,56 @@ import NetInfo from '@react-native-community/netinfo';
 
 import Navigation from './src/navigation';
 import { conn, query, tables } from './src/database';
-import { UIState } from './src/store';
+import { UIState, AuthState, UserState } from './src/store';
+import { crudSessions, crudUsers } from './src/database/crud';
 
 const db = conn.init;
 
 const App = () => {
+  const handleCheckSession = () => {
```

Comment:
is it possible to move this outside the main component (App)? I don't see any necessary callback here

Reply:
Hi mas @dedenbangkit , 
If we move this function outside a component, I think we'll not be able to update the global state value.